docs/installing-solidity.rst

```diff
@@ -60,17 +60,33 @@ Please refer to the solc-js repository for instructions.
 Docker
 ======
 
-We provide up to date docker builds for the compiler. The ``stable``
-repository contains released versions while the ``nightly``
-repository contains potentially unstable changes in the develop branch.
+Docker images of Solidity builds are available. Use the ``stable`` tag for the
+latest released version, and ``nightly`` for potentially unstable changes in the develop branch.
+
+.. code-block:: bash
+
+    docker run ethereum/solc:stable
+
+You can also specify a release build version in the tag, for example, for the 0.5.4 release.
+
+.. code-block:: bash
+
+    docker run ethereum/solc:0.5.4
+
+The Docker image contains the compiler executable, so you can pass all compiler arguments to it. For example.
 
 .. code-block:: bash
 
     docker run ethereum/solc:stable --version
 
-Currently, the docker image only contains the compiler executable,
-so you have to do some additional work to link in the source and
-output directories.
+To use the Docker image to compile Solidity files on the host machine mount a
```

Comment:
Perhaps mention that using the compiler in standard-json-io mode works just out of the box, so you can plug the command `docker run ethereum/solc:stable --standard-json` anywhere you would use `solc`.